solver/src/main/java/org/chocosolver/solver/constraints/Constraint.java

```diff
@@ -84,6 +86,11 @@
      */
     private String name;
 
+    /**
+     * If a constraint is enabled to the propagation engine.
+     */
+    private boolean enabled = Boolean.TRUE;
```

Comment:
Shouldn't it be `true`? What is the benefit of using Boolean.TRUE?

Reply:
Sorry, it's a costume that I have.
I checked on the internet and to use the primite `true` only requires a shift in registers (that's more efficient than initialize a static class). In fixed to use primitive `true`.